index.html

```diff
@@ -41,30 +41,7 @@ <h1>Crypto <br> made social</h1>
         <div class="modal-content">
             <h2 class="">Join our waitlist</h2>
             <p>We'll let you know once a spot becomes available.</p>
-            <script>
-                window.handleSubmit = (event) => {
-                    event.preventDefault();
-
-                    const myForm = event.target;
-                    const url = myForm.action;
-                    const formData = new FormData(myForm);
-
-                    const successElement = document.querySelector('#success');
-                    const errorElement = document.querySelector('#error');
-                    successElement.classList.add('dnone');
-                    errorElement.classList.add('dnone');
-
-                    fetch(url, {
-                        method: "POST",
-                        headers: { "Content-Type": "application/x-www-form-urlencoded" },
-                        body: new URLSearchParams(formData).toString(),
-                    })
-                        .then(() => successElement.classList.remove('dnone'))
-                        .catch(() => errorElement.classList.remove('dnone'));
-                }
-            </script>
```

Comment:
This probably should've worked but you were trying to POST to a value from the `action` attribute of the form but you have no `action` on the form. So it could have been `fetch("/")` instead.

Although, I'd also changed my version to use `addEventListener` instead of `onsubmit` which is the older ways to do things in HTML. Coming from React, it might seem like the way to go because of React's `onSubmit`, but that's totally different and only exists to fit the JSX markup syntax.